### ISSUE TITLE
can not support git clone --recursive when download the playbook

### ISSUE THREAD

**Issue by modoojunko** [ansible-semaphore/semaphore#180]
my playbook repo used git submodule, semaphore only did git clone and forget the "--recursive"

**Comment by ados1991** [comment]
do it manually in your repo playbook

**Comment by matejkramny** [comment]
todo: `git clone --recursive` when cloning repositories

**Comment by modoojunko** [comment]
in `api/tasks/runner.go `

here is 
```go
cmd.Args = append(cmd.Args, "clone", t.repository.GitUrl, repoName) #missing --recursive here
```

**Comment by matejkramny** [comment]
Merged with #181
